Sensus.Shared/Probes/User/Scripts/Script.cs

```diff
@@ -188,6 +199,23 @@ public Script Copy(bool newId)
             copy.Runner = Runner;
 
             return copy;
-        }
+        }
+
+        //Compare to using the DisplayDateTime.  This is used in order to determine the sort order for placement
+        public int CompareTo(Script that)
+        {
+            if (this.DisplayDateTime > that.DisplayDateTime)

```

Comment:
Would it work just as well to do `return this.DisplayDateTime.CompareTo(that.DisplayDateTime)`?

Reply:
Yes, that is the cleaner code solution.  I have made that change.